graphql/data/schema.js

```diff
@@ -2655,6 +2656,34 @@ const setUserCauseMutation = mutationWithClientMutationId({
     return setUserCause(context.user, userGlobalObj.id, causeId)
   },
 })
+
+const createSearchEnginePromptLogMutation = mutationWithClientMutationId({
+  name: 'CreateSearchEnginePromptLog',
+  inputFields: {
+    userId: { type: new GraphQLNonNull(GraphQLString) },
+    searchEnginePrompted: { type: new GraphQLNonNull(GraphQLString) },
+    switched: { type: new GraphQLNonNull(GraphQLBoolean) },
+  },
+  outputFields: {
+    user: {
+      type: userType,
+      resolve: user => user,
```

Comment:
Currently doesn't match the resolved type. This can just resolve to "success: true" like the other logs rather than a user type.

Reply:
Whoops, likely copypasta. Thanks